scipy/sparse/lil.py

```diff
@@ -111,6 +115,29 @@ def __init__(self, arg1, shape=None, dtype=None, copy=False):
                 for i in range(M):
                     self.rows[i] = []
                     self.data[i] = []
+            elif len(arg1) == 2:
+                # (data, rows) format
+                (data, rows) = arg1
+                rows_len = len(rows)
```

Comment:
to match the above case, I might replace `rows_len` with `M`. It's less readable in isolation, but I think it's useful to be consistent in context.

Reply:
Yeah, you are right.